tests/test_cli.py

```diff
@@ -6,68 +6,72 @@
 DATA = Path(__file__).parent / 'data'
 
 
+def _run(command):
```

Comment:
The changes in this module are just some refactoring mainly for not filtering the exception when an error is thrown. You can ignore it.